shared/output/gapic/templates/garbage/proto_docs/garbage/garbage.rb

```diff
@@ -201,6 +201,8 @@ class SimpleGarbageItem
       # @!attribute [rw] oneof_multiple_double
       #   @return [::Float]
       #     This is a multiple-field oneof's double field.
+      # @!attribute [rw] optional_int32
+      #   @return [::Integer]
```

Comment:
For an optional, what is the type when the field is not present? `nil`? Or does it return the default value and you have to ask `has_optional_int32?` to determine presence?

Reply:
the second option, with the default value, just like oneofs

Reply:
I wish there was a way to document that the field is optional (and thus that `has_optional_int32?` also exists to be called). But that's probably just something we can add on to #470

Reply:
added